src/fetchers/onchain.js

```diff
@@ -179,7 +176,7 @@ export const fetchTokens = async (chainId, tokens) => {
   // TODO: Add support for other chains via RPC_URL
   if (tokens.length === 0) return [];
   if (!Array.isArray(tokens)) tokens = [tokens];
-  if (!ALLOWED_CHAIN_IDS.includes(chainId)) throw new Error("Invalid chainId");
+  if (!process.env[`RPC_URL_${chainId}`]) throw new Error("Missing RPC_URL for chain");
```

Comment:
i would do it based on the network name, it would be more readble in the env var

Reply:
we have these RPC_URL_[CHAIN_ID] already in use for onchain metadata - you think we should rename them as part of this change?

Reply:
how much of an effort is it?

Reply:
updated it - ended up adding another export for the chain/network dict after all since this case also needed it in the other format. 
updating the corresponding kubernetes PR as well to have both RPC_URL formats and then will remove the old format once both changes are merged